server/src-rsr/initialise.sql

```diff
@@ -433,8 +435,13 @@ $function$
 $function$
 LANGUAGE plpgsql;
 
-CREATE TRIGGER hdb_schema_update_event_notifier AFTER INSERT ON hdb_catalog.hdb_schema_update_event
-  FOR EACH ROW EXECUTE PROCEDURE hdb_catalog.hdb_schema_update_event_notifier();
+CREATE TRIGGER hdb_schema_update_event_notifier_insert AFTER INSERT ON
```

Comment:
`INSERT` trigger is not useful when `ON CONFLICT` clause succeeds and `DO UPDATE` happens. Postgres treats it as a normal `UPDATE` statement. So, `INSERT` and `UPDATE` triggers are defined.

Reply:
Can you create an `AFTER INSERT OR UPDATE` trigger instead of creating separate `INSERT` and `UPDATE` triggers?